components/match2/wikis/callofduty/match_summary.lua

```diff
@@ -23,6 +23,11 @@ local NO_CHECK = '[[File:NoCheck.png|link=]]'
 
 local LINK_DATA = {
 	cdl = {icon = 'File:Call of Duty League Logo Small.png', text = 'Call of Duty League matchpage'},
+	breakingpoint = {
+		icon = 'File:Breaking Point GG icon lightmode.png',
+		iconDark = 'File:Breaking Point GG icon darkmode.png',
```

Comment:
could css invert instead
mediawiki:Common.css/Miscellaneous.css —> search for invert

Reply:
Since we already have both versions uploaded i think its best to just use the different images rather than CSS invert